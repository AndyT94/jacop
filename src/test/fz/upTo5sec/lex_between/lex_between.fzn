array [1..4] of var int: vector:: output_array([1..4]) = [5,2,6,2];
solve  satisfy;
