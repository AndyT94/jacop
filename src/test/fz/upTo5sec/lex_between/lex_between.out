vector = array1d(1..4, [5, 2, 6, 2]);
----------